VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
If we can specify a special case, then we must do it instead of common case. The extension is always `.VC.VC.opendb` for the case of new database engine.